packages/cli/config-management/generators-configuration/src/convertGeneratorsConfiguration.ts

```diff
@@ -91,6 +91,7 @@ async function convertGenerator({
         version: generator.version,
         config: generator.config,
         outputMode: await convertOutputMode({ absolutePathToGeneratorsConfiguration, generator }),
+        specialCasing: generator["special-casing"] ?? false,
```

Comment:
i think `smart-casing` here is probably better than `special-casing` but ik thats a really annoying change

Reply:
Let's do it, won't be too bad with some `sed`!